src/ViewHelpers/DetailsViewHelper.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace BisquidsTin\ViewHelpers;
+
+use BisquidsTin\Classes\Biscuits;
+
+class DetailsViewHelper
```

Comment:
This shouldnt be a new class, just an additional method on the biscuit view helper